phalcon/forms/element/select.zep

```diff
@@ -74,7 +74,10 @@ class Select extends Element implements ElementInterface {
 	 */
 	public function addOption(var option) -> <Element>
 	{
-		let this->_optionsValues[] = option;
```

Comment:
The question is what is the intended behavior? Doc block states an array and method signature doesn't enforce it. The behavior has changed between branch 1.x and 2.x although the signature didn't. I would personally go for an array cast, but since there are no test for it and if we want to maintain backwards compatibility, I need to change the implementation to support strings as well.